src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -14,7 +14,9 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public int matchResult(int res1, int res2, int bet1, int bet2) {
+        return (res1 == bet1 && res2 == bet2) ? (2) :
```

Comment:
```suggestion
        return (res1 == bet1 && res2 == bet2) ? 2 :
```
why is 2 in brackets?

Reply:
Просто для читабельности добавил :D